gwr/diagnostics.py

```diff
@@ -79,3 +79,11 @@ def get_CV(gwr):
     cv = np.sum(aa**2)/gwr.n
     return cv
 
+def corr(cov):
```

Comment:
any reason this doesn't just wrap `np.corrcoef?` 😃

Reply:
So if you have an intercept, I think I was just looking for a way to gracefully return a response, since it will give you a singular matrix error. Perhaps better to just exclude the intercept for this measure.